packages/core/bundle.esbuild.ts

```diff
@@ -19,18 +19,26 @@ const aliasModule = (moduleFrom: string, moduleTo: string): esbuild.Plugin => ({
   },
 });
 
-esbuild.build({
+const options: esbuild.BuildOptions = {
```

Comment:
@abrenneke I'm not sure what I was doing here. Sourcemaps are still not working for me in VSC debugger, but I was able to use the JS files to debug OK.

Reply:
Yeah I had to add this, it's fine!